src/test/kotlin/org/opensearch/commons/alerting/model/AlertContextTests.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.commons.alerting.model
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.opensearch.commons.alerting.randomAlertContext
+
+class AlertContextTests {
+    private var alertContext: AlertContext = randomAlertContext()
+
+    @BeforeEach
+    fun generateRandomData() {
+        alertContext = randomAlertContext()
```

Comment:
nit: is this required? we seem to be doing this initialization of the `var` itself?

Reply:
Removed.